js/qm-numbers-rating.js

```diff
@@ -0,0 +1,83 @@
+jQuery(document).ready(function () {
+
+    console.debug({
+        variableName: qmShortCodeDefinedVariable,
+        isNegative: qmShortCodeDefinedNegative,
+        showLabels: qmShortCodeDefinedShowLabels
+    });
+
+    if (qmShortCodeDefinedShowLabels === 'false') {
+        jQuery('.rating-button-label').hide();
+    }
+
+    var ratingButtons = jQuery('.rating-button');
+
+    var colorClasses = ['qm-black', 'qm-blue', 'qm-green', 'qm-yellow', 'qm-red'];
+
+    for (var i = 0; i < ratingButtons.length; i++) {
+        var button = ratingButtons[i];
+
+        if (qmShortCodeDefinedNegative === 'true') {
+            jQuery(button).addClass(colorClasses[colorClasses.length - i - 1]);
+        } else {
+            jQuery(button).addClass(colorClasses[i]);
+        }
+
+    }
+
+    Quantimodo.getVariableByName(qmShortCodeDefinedVariable, function (variable) {
```

Comment:
Line is too long.<br>'Quantimodo' is not defined.<br>'qmShortCodeDefinedVariable' is not defined.